db/migrations/20181114200131_create_vat_storage_tables.sql

```diff
@@ -60,10 +65,11 @@ CREATE INDEX vat_ilk_line_ilk_index
 CREATE TABLE maker.vat_ilk_spot
 (
     id        SERIAL PRIMARY KEY,
+    diff_id   INTEGER NOT NULL REFERENCES storage_diff (id) ON DELETE CASCADE,
```

Comment:
👍 Nice catch! Updated in 1ad6498. 

I'm also wondering if this sort of change is something we'd want to consider for `header_id` and `address_id`. Seems like the number of headers and addresses will grow way slower than diffs, but it would probably also be easier to handle the type change now vs. later.

Reply:
👍 totally makes sense. I'll probably do this in another PR.